### DISCUSSION TITLE
[Feature] Administration - Users and Server Stats page refactor/improvements

### DISCUSSION THREAD

**Discussion by LucaTheHacker** [immich-app/immich#11889] [Feature Request]
### The feature

Currently we have a contextual misalignment: user information about usage is shown in _Server Stats_ page rather than _Users_ page. 

Information about data usage is strictly linked to Users, due to quota settings, and since quota settings are in the _Users_ page, usage data should be there as well.

Resolving this contextual issue, we will create two additional issues to address: 
1. Too much free space in _Server Stats_ page
2. Not-beautiful displaying of information in _Users_ page

In order to fix all these issues, those steps are proposed for _Users_:
1. Move `Picture`, `Video` and `Usage` counters to _Users_ page, adding details for `Usage` for `Pictures` and `Videos`
2. Remove buttons for User edit and deletion, switch to icons-only for smaller table rows and better presentation
3. Move `Create new User` button from bottom (hard to reach when many users are present) to top, like `Create new library` for _External Libraries_.
4. Add SSO indicator in `Name`, so that we know who logs in using SSO.
5. Fix padding in Table, replacing h-80 with padding (15px 10px feels right to me)

Users page layout example:
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/cedcae52-339c-4c1f-a718-328c9fde0c25">

Regarding the now "too blank" Server Stats, I believe it would be better to add more intesting data about the usage, like:
- Counters for pictures and videos (+ size) added in the last 7, 30, 90 days
- Counter for users and users that are Immich contributors (like: Total Users: 10 | Contributors: 7)
- Network usage, to better understand how much bandwidth does Immich generate on our network (divided by local traffic and public internet traffic)
- Graphs for storage usage over time
- Add your suggetions

Along with all these upgrades, I believe some changes to the Server Stats API that reflect on the Mobile UI and Web Interface _Gallery_ must be done as well.
Currently, data usage shows data for the whole server, like "1.4TiB of 32.7TiB Used". 
In my opinion this should be divided between "used by me" and "used by others", also because as of now there is no way from the Mobile App to know how much storage is being used by the logged account. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by fuomag9** [comment]
I especially agree on the data used by the user, as it's not possible to currently understand how much space I'm using, and revealing the total space to the user is something that I'd avoid (for example imagine if a immich service provided revealed that it had 7PB total of storage to everyone )

**Comment by bo0tzz** [comment]
Coming in the next release
